packages/core/content-manager/admin/src/preview/components/PreviewContent.tsx

```diff
@@ -21,6 +21,7 @@ const PreviewContent = () => {
       height="100%"
       borderWidth={0}
       tag="iframe"
+      data-testid="preview-iframe"
```

Comment:
What went wrong without the testid? Also have you considered relying on the title instead for the selector? We can change that title for something more precise if needed

Reply:
Playwright discourages the use of waitForSelector. Good idea I'l getByTitle